### DISCUSSION TITLE
milestone 3-1

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#307] [General]
Weekly meetings, scrump prep, issue validations

**Comment by PinkSylvie** [comment]
This week I assisted in the work for the milestone 2 presentation and I prepared the tasks that I will be working on this incoming milestone

**Comment by fernandopizarro1** [comment]
Finished AdvancedBot Implementation, Removed gameState placeholder to the lobby render. Started working on Lecture Topics.

**Comment by AngelCIICMorales** [comment]
Held weekly meetings and helped the completion of the presentation.

**Comment by ebrain-mendoza121** [comment]
Attended weekly meetings, discussed goal for milestone 3 and starting to prepare task to work on this milestone.

**Comment by RicardoJRSs** [comment]
Prepared tasks for milestone and reviewed the milestone guidelines.

**Comment by Adriana-Isabel** [comment]
Attended meetings, searched for ideas for the Lecture Topic Tasks and brainstormed ideas for a task in the Home Page.

**Comment by Gaby-Concepcion** [comment]
Worked on video presentation for the milestone 2 retrospective, attended weekly meeting and opened some issues for the upcoming final milestone.

**Comment by sandy-marrero** [comment]
Attended weekly meeting, started research on documentation tasks for this semester.

**Comment by rukitoyordan** [comment]
Gathered information for new LTTs and Milestone Tasks

**Comment by ilikecheese117** [comment]
Studying to find out what lecture topic tasks I can do.

**Comment by nsfrontera** [comment]
Working on creating and assigning task for the sprint

**Comment by print-Eruki** [comment]
Creating more tasks for milestone 3 based on the goals for the milestone

**Comment by AnthonyMed13** [comment]
Created task for milestone 3 and research other possible task

**Comment by JadiellMV** [comment]
Researched for tasks that will help achieve the milestine 3 goals.

**Comment by Negroni1** [comment]
Finished the milestone 2 presentation slides and work on drafting milestone 3 tasks

**Comment by Joaquin939** [comment]
This week I began to work on creating my issues for this milestone

**Comment by G4vp** [comment]
Work on drafting Milestone 3 Lecture Topic Tasks

**Comment by FrancesC8** [comment]
Working on identifying the scope of Milestone 3 tasks.

**Comment by Jvstinoo** [comment]
Began working on creating lecture topic tasks.

**Comment by ianramospr** [comment]
Drafting my issues for this milestone

**Comment by Saul-Caban-UPRM** [comment]
Brainstorming task for milestone 3 deciding what LTT to do

**Comment by frances-sepulveda** [comment]
Stated working with lecture topic tasks missing for this milestone
